src/modules/wallet/components/overview/overview.js

```diff
@@ -1,73 +1,73 @@
 /* eslint-disable max-statements */
-import React, { useEffect } from 'react';
-import { useSelector, connect } from 'react-redux';
-import { withRouter } from 'react-router';
-import { compose } from 'redux';
-import { withTranslation } from 'react-i18next';
+import React, { useCallback, useEffect, useMemo } from 'react';
+import { useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
 import grid from 'flexboxgrid/dist/flexboxgrid.css';
-
-import withData from 'src/utils/withData';
-import { getTransactions } from '@transaction/api';
+import { useTokensBalance } from 'src/modules/token/fungible/hooks/queries';
+import TokenCard from 'src/modules/wallet/components/TokenCard';
+import TokenCarousel from 'src/modules/wallet/components/TokenCarousel/TokenCarousel';
 import { selectActiveTokenAccount } from 'src/redux/selectors';
 import FlashMessageHolder from '@theme/flashMessage/holder';
 import WarnPunishedDelegate from '@dpos/validator/components/WarnPunishedDelegate';
-import WalletInfo from '@wallet/components/walletInfo';
-import BalanceInfo from '@token/fungible/components/BalanceInfo';
+import WalletVisualWithAddress from '@wallet/components/walletVisualWithAddress';
+import DialogLink from 'src/theme/dialog/link';
+import { useCurrentAccount } from '@account/hooks';
+import { useBlocks } from '@block/hooks/queries/useBlocks';
+import { SecondaryButton, PrimaryButton } from '@theme/buttons';
+import { useDelegates } from '@dpos/validator/hooks/queries';
+import { selectSearchParamValue } from 'src/utils/searchParams';
+import { useAuth } from '@auth/hooks/queries';
 import styles from './overview.css';
+import chainLogo from '../../../../../setup/react/assets/images/LISK.png';
 
-const mapStateToProps = (state) => ({
-  currentHeight: state.blocks.latestBlocks.length
-    ? state.blocks.latestBlocks[0].height
-    : 0,
-});
+// 6: blocks per minute, 60: minutes, 24: hours
+const numOfBlockPerDay = 24 * 60 * 6;
```

Comment:
Please add a TODO, we need to remove this calculation logic in UI and SDK exposes DPoS endpoint to get the exact time details.